### ISSUE TITLE
Diplomatic transcriptions. 

### ISSUE THREAD

**Issue by jcowey** [jcowey/P3#21]
I had thought of using `<quote type="edition">` for these. That gets rid of the "Edition" marker in heiEditions HTML.

I now prefer the idea of declaring the diplomatic transcript for what it is. I am imagining in greyed lettering 

"Diplomatic transcription" or the like. 

Am hoping that this should not involve to much reengineering. Use `<div copyOf="#ed2" type="edition"/>` in the XML; have the diplomatic transcription encoded in EpiDoc XML (as we have it for benaissa_annunciation); find out how the UB HD can display what we want to see - possible a subtype on the `<div>`

**Comment by samosafuz** [comment]
I agree that "Diplomatic transcription" is the optimal text to display in the article, in greyed lettering. 

Managing the xml should not be unduly complicated. The most straightforward way, IMO, is to declare `@type="edition" @subtype="diplomatic"` in a `<div>` element, and to encode the diplomatic transcription in EpiDoc, as you suggest. 

Drilling down from there, your first instinct, to use `<quote type="edition">` is correct, IMO, for including the EpiDoc transcription within the article itself. That will simplify things: if we declare the diplomatic transcription in this way, there is no need to distinguish between `#ed1` and `#ed2` and to clutter up the xml file with a separate, largely redundant edition (which, in DCLP, will come with reduplicated metadata, too). Only the articulated version needs to be processed into a separate xml segment and to be called by `#ed1`, since only the articulated version will end up in PN. 

The task for the UB will be to transform `@subtype="diplomatic"` like a typical edition, but with the modified header.

**Comment by jcowey** [comment]
One of the reasons that I would slightly prefer to keep the "sort of redundant" `#ed1` is that the encoding of underdots is much more properly controlled. Fiddling with whitespace to get good display of underdots is something I would prefer to minimise.

**Comment by samosafuz** [comment]
Yes, that's right: apologies. We call them in Pylon as follows:
```
<div copyOf="#ed1" type="edition" subtype="diplomatic"/>
<div copyOf="#ed2" type="edition"/>
<div copyOf="#trans1" type="translation"/>
```
This is much better: there's no need for `@subtype="diplomatic"` in the individual XML segment, but only when we call it via `@copyOf`.

**Comment by jcowey** [comment]
See now: https://gitlab.ub.uni-heidelberg.de/verlag/PapyrologicalPublicationPlatform/-/issues/120
